web/src/components/tasks/business-formation/review/ReviewSignatures.tsx

```diff
@@ -19,9 +20,43 @@ export const ReviewSignatures = (): ReactElement => {
     };
   };
 
+  const displayEmptySignersIfNotEntered = (): ReactElement | undefined => {
```

Comment:
I might recommend moving the if-statement out of this and into the place where this function gets called. Having an "if" in the name of the method is a little confusing, not least because it doesn't fully capture the extent of the "if" statement in here. Might be cleaner to name the function `displayEmptySigners` and move the if-statement to the render

Reply:
It would have the added benefit of always returning a thing, never undefined

Reply:
I was debating between the two for the same reasons but decided to go with this because it felt cleaner in the code but agree with your points regarding the confusion it might lead to, making the change.

Reply:
looking at the code below, there's a lot going on with the mapping of signers and incorporators, and then these functions in there as well. What if we have a strcuture like this? (psuedocode)

```
const signers = () => {
if (hasNoSigners) {
  return displayEmptySigners()
 } else {
  return displaySigners()
 }
}

const incorporators = () => {
 if (hasNoIncorporators) {
  return displayEmptyIncorporators()
 } else {
  return displayIncorporators()
 }
}

const displayEmptyIncorporators = () => <>...</>
const displayEmptySigners = () => <>...</>
const displayIncorporators = () => <>...</>
const displaySigners = () => <>...</>


render (
  {signers()}
  {incorporators()}
)

```

Reply:
I like this, makes it easier to read. I will restructure it.